test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/AddressRegistrationTests.cs

```diff
@@ -147,13 +148,14 @@ public class AddressRegistrationTests
             {
                 host.Start();
 
-                var debugLog = testLogger.Messages.Single(log => log.LogLevel == LogLevel.Debug);
-                Assert.True(debugLog.Message.Contains("default"));
+                Assert.Equal(5000, host.GetPort());
+                Assert.Single(testLogger.Messages.Where(log => log.LogLevel == LogLevel.Debug &&
+                    string.Equals($"No listening endpoints were configured. Binding to {Constants.DefaultIPEndPoint} by default.",
+                    log.Message, StringComparison.Ordinal)));
 
-                foreach (var testUrl in new[] { "http://127.0.0.1:5000", "http://localhost:5000" })
+                foreach (var testUrl in new[] { "http://127.0.0.1:5000", /* "http://[::1]:5000" */})
```

Comment:
We need to add it back soon. Similar to https://github.com/aspnet/KestrelHttpServer/commit/4533383612fccc324337528f50ce324d7fc10d27.